Helpers/opent2t-translator-helper-hue/js/index.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
```

Comment:
Agreed, you should have two examples (nest hub and wink hub) to follow.

---
In reply to: [78867509](https://github.com/openT2T/translators/pull/96#discussion_r78867509) [](ancestors = 78867509)